BlackLagerMessage.proto

```diff
@@ -1,5 +1,47 @@
 syntax = "proto3";
 
+/*
+ * https://grpc.io/docs/protoc-installation/
+ * Useful protc installation guide for reference
+ */
+
+/*
+ * Can contain signed message or Persona protobuf
+ */
+message BlackLagerMessage{
+
+  /*
+  * Used to differentiate text or persona
+  */
+  int32 type = 1;
```

Comment:
Using a 32-bit integer for a binary flag is space inefficient. This could be a boolean "isMessage". Such a flag may not even be necessary. The receiver can check if a BlackLagerText or Persona exists.